pallets/protos/rpc/runtime-api/src/lib.rs

```diff
@@ -10,7 +10,7 @@ sp_api::decl_runtime_apis! {
 		AccountId: Codec
 	{
 
-		fn get_by_tags(tags: Vec<Tags>, owner: Option<AccountId>, limit: u32, from: u32, desc: bool) -> Vec<Hash256>;
+		fn get_by_tags(tags: Vec<Tags>, owner: Option<AccountId>, limit: u32, from: u32, desc: bool) -> Option<Vec<Hash256>>;
```

Comment:
Empty vector not enough :) ?

Reply:
I thought we only wanted "empty elements" for `get_metadata_batch()` ?

What do you mean by **"Empty vector not enough :) ?"** ?

Reply:
Just return an empty vector if there are not results. Why we return Option? It's pointless.

Reply:
True. Changed in latest commit